hack/lib.sh

```diff
@@ -47,7 +51,6 @@ function hack::verify_terraform() {
 }
```

Comment:
I am wondering why we have terraform here? Do we need to remove that?

Reply:
removed now, we won't use terrform currently.